app/views/logged_in.erb

```diff
@@ -0,0 +1,10 @@
+<span id='logged_in'>
+  <form action='/sessions/<%= session[:user] %>' method='post'>
+    <b>You're Logged In As: <%= session[:name] %>!</b>
```

Comment:
I don't like seeing informational stuff not directly related to `form`s inside a `form`. Put this outside.